src/hoprd/hoprd_deployment.rs

```diff
@@ -464,22 +470,27 @@ fn build_env_vars(identity_pool: &IdentityPool, identity_hoprd: &IdentityHoprd,
         value: Some(session_port_range),
         ..EnvVar::default()
     });
+    env_vars.push(EnvVar {
+        name: constants::HOPRD_DEFAULT_SESSION_LISTEN_HOST.to_owned(),
+        value: Some(format!("{}:0", hoprd_host)),
```

Comment:
The variable `hoprd_host` contains the public IP of the exposed load balancer service, is not the internal IP of the POD.
That change was already added in this PR: https://github.com/hoprnet/hoprd-operator/pull/258/files#diff-c6147f00ae7d5cc8d4e077dae522b1734788459bf23ad00d2625ecb3ef628123

And then reverted in this commit: https://github.com/hoprnet/hoprd-operator/commit/c4c527022b59170fca4fae25af054cb56dfb26ea#diff-c6147f00ae7d5cc8d4e077dae522b1734788459bf23ad00d2625ecb3ef628123

I need to understand better the system requirements of the node, so we incorporate those changes into the hoprd-operator.